packages/shared/src/components/cards/ArticlePostCard.tsx

```diff
@@ -47,6 +48,7 @@ export const ArticlePostCard = forwardRef(function PostCard(
   const customStyle = !showImage ? { minHeight: '15.125rem' } : {};
   const { showFeedback } = usePostFeedback({ post });
   const isFeedPreview = useFeedPreviewMode();
+  const isVideoType = post.type === PostType.VideoYouTube;
```

Comment:
I noticed we have multiple definitions of `isVideoType` sometimes it's like this, sometimes it also checks if the sharedPost is a video.
Wondering if it will cause confusion down the line, and maybe we should turn this into a utility functions like we do to determine the sharedPost for instance?
It could take a prop of `alsoCheckSharedTyped` (suck at naming)

Reply:
I think just checking like this works. Kinda what I did in #2434 (at first, I see I removed it)

```js
post.type === PostType.VideYouTube ||
(post.type === PostType.Shared && post.sharedPost.type === PostType.VideoYouTube)
`

Reply:
@rebelchris refactored the check if post is youtube or shared youtube to a dedicated util function in b7794e1